vnext/Shared/WinRTWebSocketResource.cpp

```diff
@@ -185,13 +185,10 @@ fire_and_forget WinRTWebSocketResource::PerformPing() noexcept
   }
 }
 
-fire_and_forget WinRTWebSocketResource::PerformWrite() noexcept
+fire_and_forget WinRTWebSocketResource::PerformWrite(string&& message, bool isBinary) noexcept
 {
   auto self = shared_from_this();
-  if (self->m_writeQueue.empty())
-  {
-    co_return;
-  }
+  auto messageLocal = std::move(message); // Persist message through coroutine resumptions
```

Comment:
>messageLocal [](start = 7, length = 12)

Why do we need this extra variable? just use "message" instead.

Reply:
Because of coroutine behavior.
`message` was not referenced until lines 212/221.
That is, after the third coroutine resumption. If no reference is held, the `message` value won't be available to the resumption point's frame.
Hence, test `SendConsecutive` hangs.